client/my-sites/plugins-wpcom/plugin-install-button.jsx

```diff
@@ -16,6 +16,7 @@ import { getSelectedSite } from 'state/ui/selectors';
 import { getEligibility } from 'state/automated-transfer/selectors';
 import { initiateThemeTransfer } from 'state/themes/actions';
 import { transferStates } from 'state/automated-transfer/constants';
+import analytics from 'lib/analytics';
```

Comment:
For connected components we prefer to use the [redux analytics actions](https://github.com/Automattic/wp-calypso/tree/master/client/state/analytics) instead of the analytics lib directly. This has two benefits:
1. Tests are easier to create since lib/analytics assumes window is available
2. If we trigger a redux action, we can compose some meta data on the action in order to trigger the analytics middleware